backend/src/email/models/email-message-transactional.ts

```diff
@@ -37,7 +37,7 @@ export class EmailMessageTransactional extends Model<EmailMessageTransactional>
     primaryKey: true,
     allowNull: false,
   })
-  id: number
+  id: string
```

Comment:
this one can just change without any db migration? 😯

Reply:
The previous data type declaration, number, is wrong. Number in JS can't fit SQL BIGINT

Reply:
Ohh I see, makes sense!